Marketplace/Marketplace Updates/Marketplace Updates July 2017.md

```diff
@@ -12,7 +12,7 @@ The Marketplace is constantly evolving with more products, providers, and offeri
 Products added to the CenturyLink Marketplace July 2017 include:
 
 
-* [*OSNEXUS QuantaStor SDS EE with Gold Support*](https://www.ctl.io/marketplace/partner/ZV5T/product/QuantaStor%20SDS%20with%20Gold%20Support%20%28License%20Only%29/) - QuantaStor is a unified Software Defined Storage (SDS) platform designed to scale out to make storage management easy while reducing overall enterprise storage costs. This is a License Only purchase. The customer is responsible for deploying QuantaStor SDS separately on the infrastructure of their choosing.
+* [*OSNEXUS QuantaStor SDS EE with Gold Support*](https://www.ctl.io/marketplace/partner/ZV5T/product/QuantaStor%20SDS%20with%20Gold%20Support%20%20(License%20Only)/) - QuantaStor is a unified Software Defined Storage (SDS) platform designed to scale out to make storage management easy while reducing overall enterprise storage costs. This is a License Only purchase. The customer is responsible for deploying QuantaStor SDS separately on the infrastructure of their choosing.
```

Comment:
Change open paren `(` to `%28`, change ')' to `%29`
Otherwise the link will not be parsed correctly by the KB processor.